src/components/PaymentLog.vue

```diff
@@ -641,6 +640,7 @@ export default {
       if (this.eligible && renewal) {
         data.discount = [renewal];
       }
+      this.salesLogForm.repayment_cycle_id?.id == 3 ? data.custom_date = this.salesLogForm.custom_date : ''
```

Comment:
what I was expecting is Number(this.salesLogForm.custom_date)

Reply:
it fails if other bimonthly or monthly is chosen. so i had to add it with condition